testdata/ostrusted/InterfaceProblem.java

```diff
@@ -0,0 +1,9 @@
+//@skip-test
+interface Interface {
+
+}
+
+class InterfaceProblem implements Interface {
+	Interface i = this;
```

Comment:
Please use spaces rather than a tab.
One should never use tabs in code files. They display differently in different editors, and they often print differently than they display in an editor. Always use spaces instead.